hack/jenkins/e2e.sh

```diff
@@ -937,8 +937,6 @@ case ${JOB_NAME} in
     : ${E2E_TEST:="false"}
     : ${E2E_DOWN:="false"}
     : ${NUM_MINIONS:=5}
-    : ${ENABLE_DEPLOYMENTS:=true}
```

Comment:
I assume these is removed because the current latest release doesn't support them?